ui/src/shared/components/Dropdown.js

```diff
@@ -143,86 +141,19 @@ class Dropdown extends Component {
       <ul
         className={classnames('dropdown-menu', {
           'dropdown-menu--no-highlight': useAutoComplete,
+          [menuClass]: menuClass,
```

Comment:
This one does not have a default value, but I agree there has to be a better way